test-server/app.py

```diff
@@ -1,6 +1,18 @@
 
+import os
+
+
+if os.environ.get('NEW_RELIC_LICENSE_KEY'):
+    try:
+        import newrelic.agent
+        newrelic.agent.initialize(os.path.abspath(os.path.join(os.path.dirname(__file__), 'newrelic.ini')))
+    except ImportError:
+        pass
+
```

Comment:
yep this should be probably in the other place where I've added this - here it's only for e2e test server